<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddFeed</class>
 <widget class="QDialog" name="DialogAddFeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add feed</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSource">
       <property name="text">
        <string>Add to source :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxSource"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelURL">
       <property name="text">
        <string>Feed/website URL :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelAddToFolder">
       <property name="text">
        <string>Add to folder :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxFolder"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDiscoveredFeeds">
       <property name="text">
        <string>Discovered feeds :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ZapFR::Client::TableViewPaletteCorrected" name="tableViewDiscoveredFeeds">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditURL"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDiscover">
         <property name="text">
          <string>Discover feeds</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelDiscoveryInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter either a direct link to a feed into the text input above, a link to a website for which you want to discover existing feeds, or a YouTube channel page. Then click the 'Discover feeds' button, and pick which feed(s) you want to import from the table below.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDummy">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZapFR::Client::TableViewPaletteCorrected</class>
   <extends>QTableView</extends>
   <header>widgets/TableViewPaletteCorrected.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxSource</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddFeed</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddFeed</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
